xcode/Subconscious/Shared/Models/HeaderSubtext.swift

```diff
@@ -10,14 +10,19 @@ import Foundation
 struct HeaderSubtext: Hashable, CustomStringConvertible {
     var headers = Headers()
     var body: String
-
+    
     var description: String {
         "\(headers.toHeaderString())\(body)"
     }
-
+    
     func toMarkup() -> String {
         self.description
     }
+
+    /// Calculate size of text, including inlined headers
+    func size() -> Int? {
```

Comment:
Had to add this for test. Our actual code-path in the app re-uses the size returned by reading the file metadata. However, we can also calculate it via the string.

This will also introduce the possibility of simplifying the `writeMemo` signature for local content.